docs/src/main/tut/faq.md

```diff
@@ -22,6 +22,7 @@ position: 40
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [What is `tailRecM`?](#tailrecm)
  * [What does this symbol mean?](#symbol)
+ * [How can I test `laws`?](#law-testing)
```

Comment:
A more specific way to say could be "how can I test instances against their type classes'  laws? "

Reply:
Perfect!